### ISSUE TITLE
Let's try it out a little bit harder and make Scipy Lib work perfectly in pythonista?

### ISSUE THREAD

**Issue by kerrl** [ywangd/stash#350]
Hello:
I like pythoninsta very much. It allows me to learn python on ios. I am currently learning data analysis, so I try to install scipy lib. When I try run stash in python3.6, and try PIP install scipy, it boosts my installation success. I can see that I successfully installed scipy (1.2.0). Using the PIP list, I can see that I successfully installed scipy (1.2.0). I immediately creat test_scipy.py ,example:
  import scipy ;
  help(‘modules’)
it’s run sucessed.
I can use scipy function.
which shows that the scipy library is loaded.
However, when I restart pythonista and run test_scipy.py again, the instructions begin with an Error: "ImportError:Error importing scipy: you cannot import scipy while being in scipy source directory;Please exit the scipy source tree first,and relaunch your python interpreter.
Strange that I've used the scipy library successfully. Why is this happening?

**Comment by jsbain** [comment]
You cannot use scipy in pythonista, as it depends on c code.

**Comment by ColdGrub1384** [comment]
The problem is not C code (only @omz could implement it) but it's Fortran code that is a lot harder to make work on iOS.

**Comment by SamyBencherif** [comment]
> The problem is not C code (only @omz could implement it) but it's Fortran code that is a lot harder to make work on iOS.

Are you saying scipy runs Fortran code?

**Comment by ColdGrub1384** [comment]
@SamyBencherif Yes. I implemented NumPy, Pandas and Matplotlib successfully  in my app, but I didn’t manage to compile SciPy because of Fortran code. It’s technically possible, but very tricky. The console says ARM64 is not supported by the Fortran compiler. But there seems to be a GCC plugin to compile Fortran for iOS: https://apple.stackexchange.com/questions/40471/is-there-a-fortran-compiler-emulator-for-ios
